crates/vhost-user-backend/src/lib.rs

```diff
@@ -12,9 +12,7 @@ use std::fmt::{Display, Formatter};
 use std::path::Path;
 use std::sync::{Arc, Mutex};
 use std::thread;
-
```

Comment:
Maybe accidentally happened during the rebase, but we enforce to keep that groups separated.